server/service/src/sync/program_schemas/hiv_care_encounter.json

```diff
@@ -332,23 +287,21 @@
               "type": "number"
             },
             "tpha": {
-              "enum": [
-                "Pos",
-                "Neg"
-              ],
+              "enum": ["Pos", "Neg"],
               "type": "string"
             }
           },
           "type": "object"
         },
+        "note": {
+          "description": "An overall note for additional comments about the encounter",
+          "type": "string"
```

Comment:
Can we please use a notes array here (same as for patients). Its easy to use `notes[0]` now but its much harder to add support for multiple notes if at a later stage

Reply:
Yeah, okay, have made it into a `string[]`, and just used the first one. We don't want the full Note fields, just a simple text block.

f4c740db7

(Plus some other fixes to update the Patient type and fields)

Reply:
Lets please have the full `Note`s object then we have the same type for patient and encounter and  potentially enrollments.  An object is easy to extend while a string[] isn't. The generic base type can't easily be changed in the future so we have to plan ahead and make is easy to extend.

In the Note interface we actually don't need all the properties as required, only the text is really required. Then it becomes very easy to use here as well.

Reply:
Okay sure c68036f22